examples/arNFT_gltf_brave_robot_example.html

```diff
@@ -21,6 +21,8 @@
     <script>
 			ARnft.ARnft.init(640, 480, "examples/DataNFT/pinball", 'config_brave_robot.json', true)
 			.then((nft) => {
+				let renderer = nft.renderer;
+				console.log(renderer);
```

Comment:
This print null as in the constructor. I think we need to send the renderer object with an Event?